tools/autograd/derivatives.yaml

```diff
@@ -497,6 +497,9 @@
 - name: frac(Tensor self) -> Tensor
   self: grad
 
+- name: frexp.Tensor(Tensor self) -> (Tensor mantissa, Tensor exponent)
+  self: grad * 1 / exponent.exp2()
```

Comment:
cc @albanD could you help check this derivative for the ``torch.frexp`` function.

Reply:
That looks good.
You can remove the `1` though and just use `grad / exponent.exp2()`. But otherwise should be good.

Reply:
@albanD Thank you so much for the kind suggestion, I will update this derivative accordingly.